covalent_slurm_plugin/slurm.py

```diff
@@ -549,11 +569,11 @@ async def run(self, function: Callable, args: List, kwargs: Dict, task_metadata:
 
         if "output" not in self.options:
             self.options["output"] = os.path.join(
-                self.remote_workdir, f"stdout-{dispatch_id}-{node_id}.log"
+                self._current_remote_workdir, f"stdout-{dispatch_id}-{node_id}.log"
```

Comment:
Discussed over Slack. We will keep this one as-is. And I think that's a good decision because a lot of code that I use actually relies on the stdout and stderr being in the current working directory.